provision/acc_provision/templates/aci-operators-crd.yaml

```diff
@@ -10,4 +10,22 @@ spec:
     plural: acicontainersoperators
     singular: acicontainersoperator
   scope: Namespaced
-  version: v1alpha1
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    schema:
+      openAPIV3Schema:
+        description: CRD to own the lifecycle of ACI objects in the cluster
+        properties:
+          spec:
+            description: Spec required for a ACI CNI CRD
+            properties:
+              flavor:
+                type: string
+              config:
+                type: string
+            type: object
+        required:
+        - spec
+        type: object
```

Comment:
I don't see a status field in all of the changes like below:
          status:
            description: AciContainersOperatorStatus defines the successful completion of AciContainersOperator
            type: object

Reply:
Added the status field